### DISCUSSION TITLE
How to change "Made with MyST" at the top of the webpage

### DISCUSSION THREAD

**Discussion by raghu-mahajan** [executablebooks/meta#1110] [Q&A] [answered]
I am a researcher in theoretical physics trying to build a website using MyST for aggregating the know how that is important for my research group. I was able to install MyST and get it running on my laptop using the default "book-theme." That said, I am a complete novice in modern web design and I am unable to figure out how to, for example, remove the text "Made with MyST" at the top left of the website. I would also like to remove the "Learn More" at the top right of the page. (See the attached screenshot for more details.) 

Any help would be much appreciated! Thank you.

<img width="1352" alt="Screenshot 2023-12-03 at 5 41 22 PM" src="https://github.com/executablebooks/meta/assets/17094269/8940479f-7e66-4f6e-9807-1efde682698d">

**Comment by rowanc1** [comment, marked as answer]
Hi @raghu-mahajan, you can set the site title, logo and any navigation actions in the myst.yml inside of the 'site' dictionary.

If you don't want any actions (the learn more action), you can remove that entirely.

Hope that helps!

**Reply by raghu-mahajan** [reply]
Thank you very much @rowanc1 for the quick response! I hadn't realized that the "Made with MyST" was appearing in place of a logo. I was able to fix both the issues.

**Reply by rowanc1** [reply]
(on my phone and haven't tested this)

Can you try 'logo_text' in the site?

https://github.com/simpeg/tle-finitevolume/blob/main/myst.yml#L74

I think that if there is no logo and no logo_text, it falls back to Made with MyST, which we might want to revisit.

I am also surprised I couldn't find docs on this as most frontmatter is auto documented now! I will open an issue to add some docs. Issue is here: https://github.com/executablebooks/mystmd/issues/795

**Reply by raghu-mahajan** [reply]
I see. Indeed, `logo_text: hep-th` worked to give me "hep-th" in place of "Made with MyST".

**Reply by rowanc1** [reply]
Glad that worked! 🚀 You can also add a 'logo' with a relative path to an image.
